coreels/settings.py

```diff
@@ -91,8 +92,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
Consider using environment variables for database connection settings.

This will avoid leaking sensitive database connection strings to public repositories as this.